instances/bootstrap.treasury-factory.near/widget/config/data.jsx

```diff
@@ -1,6 +1,9 @@
-const sputnikAccount =
-  context.widgetSrc?.split("/")[0].split(".near")[0] ??
-  "testing-astradao" + "sputnik-dao.near";
+const widgetSrc = context.widgetSrc
```

Comment:
it could be worth considering to be able to reference another base config file here. so that this could only contain instance specific overrides. we could discuss this for a feature request in the future.

Reply:
sorry I don't understand, what do you mean?